src/App.js

```diff
@@ -9,9 +15,27 @@ class App extends React.Component {
 
   render() {
     return (
-      <div className="App">
-        <Calculator />
-      </div>
+      <Router>
+        <Switch>
+          <div className="App">
+            <div className="Header-Container">
+              <h1>Math Magicians</h1>
+              <nav>
+                <ul className="Nav-Items">
+                  <li className="Nav-Item"><Link to="/">Home</Link></li>
+                  <span className="Nav-Item">|</span>
+                  <li className="Nav-Item"><Link to="/calculator">Calculator</Link></li>
+                  <span className="Nav-Item">|</span>
+                  <li className="Nav-Item"><Link to="/quote">Quote</Link></li>
+                </ul>
+              </nav>
+            </div>
+            <Route path="/" exact component={Home} />
```

Comment:
- [ ] **OPTIONAL**
I highly recommend that you manage to make the application opens on the home page directly without the need of clicking the home page at first 🙏 😄 This will improve the UX of your website.

Reply:
@MahmoudBakr23 I have made this change. Thank you for the pointer.